base_set/cards/harbinger.py

```diff
@@ -0,0 +1,49 @@
+from core.card import CardType
+from core.card import KingdomCard
+from core.card_stack import StackPosition
+from core.counters import CounterId
+from core.counters import CounterName
+from core.decision import Decision
+from core.events import CardEventType
+from core.locations import Location
+from core.locations import LocationName
+
+
+class HarbingerDecision(Decision):
+    def __init__(self, game_state):
+        player = game_state.get_current_player_name()
+        cards = list(game_state.get_location(Location(player, LocationName.DISCARD)))
+        self.options = cards
+        self.min = 0
+        self.max = 1
+
+class HarbingerCard(KingdomCard):
+    types = (CardType.ACTION,)
+    base_cost = 3
+
+    @classmethod
+    def play(cls, game_state):
+        """
+        +1 card
```

Comment:
Nice, we should probably start doing this. Maybe we should make it an actual property of the Card so some agents can choose to print it